### PR TITLE
fix: when scrolled, remove blur on nav bar

### PR DESCRIPTION
**Pull request by lynnshaoyu**
https://uniswaplabs.atlassian.net/browse/WEB-1565?atlOrigin=eyJpIjoiMmRlM2MzYWM2NGMyNGIyMGE4MmYwMGZiNzA2MzA0NGUiLCJwIjoiaiJ9

### CODE REVIEW

File: src/components/WalletDropdown/AuthenticatedHeader.tsx [hunk 1]
```diff
@@ -139,8 +139,8 @@ const AuthenticatedHeader = () => {
 
   return (
     <AuthenticatedHeaderWrapper>
-      <HeaderWrapper>
-        <StatusWrapper>
+      <HeaderWrapper className="header-wrapper">
+        <StatusWrapper className="something">
```
Comment (gbugyis) [inline src/components/WalletDropdown/AuthenticatedHeader.tsx:143]: are these classNames necessary?

**Review by gbugyis — CHANGES_REQUESTED** [review]

File: src/pages/App.tsx [hunk 2]
```diff
@@ -65,11 +65,15 @@ const BodyWrapper = styled.div`
   `};
 `
 
-const HeaderWrapper = styled.div`
+const HeaderWrapper = styled.div<{ scrolledState?: boolean }>`
```
Comment (cbachmeier) [inline src/pages/App.tsx:68]: We only want this scroll behaviour on NFT Pages not all pages

**Review by cbachmeier — CHANGES_REQUESTED** [review]

File: src/pages/App.tsx (hunk 2 above)
Comment (lynnshaoyu) [reply src/pages/App.tsx]: switched to be gated by nft flag / phase 1 enabled, as discussed in slack.

File: src/components/WalletDropdown/AuthenticatedHeader.tsx (hunk 1 above)
Comment (lynnshaoyu) [reply src/components/WalletDropdown/AuthenticatedHeader.tsx]: oops my comment didnt get saved earlier - they totally are not necessary lol, thank you for catching

**Review by gbugyis — APPROVED** [review]

**Pull request merged**